prediction_market_agent/agents/known_outcome_agent/known_outcome_agent.py

```diff
@@ -42,6 +43,22 @@ def has_known_outcome(self) -> bool:
         return self.result is not Result.UNKNOWN
 
 
+HAS_QUESTION_HAPPENED_IN_THE_PAST_PROMPT = """
+The current date is {date_str}. Your goal is to assert if a QUESTION references an event that is already finished (according to the current date and time) or if it will still take place in a later date. 
+
+For example, you should return 1 if given the event "Will Bitcoin have reached the price of $100 by 30 March 2023?", since the event ends on a data prior to the current date.
+
+Your answer MUST be an integer and follow the logic below:
+- If the event is already finished, return 1
+- If the event has not yet finished, return 0
+- If you are not sure, return -1
+
+Answer with the single prompt only, and nothing else.
```

Comment:
```suggestion
Answer with the single 1, 0 or -1 only, and nothing else.
```

Just a terminology alert: `prompt` is what goes into the model, so the answer isn't a prompt, that's a completion.

Reply:
Thanks, adjusted.